### PR TITLE
publishing translation of introduction to map warper

### PR DESCRIPTION
**Pull request by hawc2**
@svmelton hopefully this one goes a little smoother

### Checklist

- [ ] Assign yourself in the "Assignees" menu
- [ ] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [ ] Add the appropriate "Label"
- [ ] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [ ] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by hawc2** [comment]
@svmelton actually let's hold off on merging this. I got the basic stuff set up, but I discovered that some of the liquid syntax isn't properly rendering the gifs and other images, so I'll want to figure this out before we merge and publish

File: en/lessons/introduction-map-warper.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+## Practice
+
+1. Click on the Rectify tab.
+2. Take a moment to move the map on the right to the North Carolina and South Carolina region. The arrows at the top of the screen move the map slightly to the north, south, east, and west, and are useful when you need to make small adjustments to the map. You can zoom in and out with the slider or with your trackpad/mouse. To move around a map, click the hand icon.
+3. Once you feel comfortable moving around the maps, select your first control point. Start from the historic map and choose a location &mdash;for example, a city&mdash; that will be relatively easy to find.
+4. Then, click the green control point marker on the modern map and find the same location to match them up.
+
+{% include figure.html filename="mapwarper_controlpointsadded.png" caption="Match up your control points." %}
+
+5. If you do not click the Add Control Point button, the next time you click on a map, the control point you've added will move. This functionality gives you the flexibility to adjust your points while adding them, but can be confusing if you don’t realize that your point has moved because you didn’t click Add Control Point.
```
Comment (svmelton) [inline en/lessons/introduction-map-warper.md:181]: This (and several points after) is appearing as #1 in the preview

File: en/lessons/introduction-map-warper.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+{% include figure.html filename="mapwarper_satellite.png" caption="Georeferenced map in satellite view." %}
+
+11. Click the Preview tab for a larger view of the georeferenced map. Changing the transparency using the slider can give you a sense of how accurate your georeferencing has been applied.
+
+{% include figure.html filename="mapwarper_preview.png" caption="Map Warper Preview." %}
+
+## Cropping
+The **Crop** tab allows you to crop a section of the uploaded map. This is useful for dividing maps into composites. The display window integrates the following actions:  
+
+* {% include figure.html filename="move_around_map.gif" %}:  **Move around Map**
```
Comment (svmelton) [inline en/lessons/introduction-map-warper.md:214]: I don't think we need these as bullet points

**Review by svmelton — COMMENTED** [review]
Thanks @hawc2—I made a couple of comments about formatting issues (they may be the same things you noticed), but once you have those fixed & the DOI added, I think it looks good.

**Comment by hawc2** [comment]
thanks @svmelton. on this one i might need to wait for @anisa-hawes to return to make sure we're in agreement about the right way to link the gifs . . .

**Comment by anisa-hawes** [comment]
Hello @hawc2. Thank you for asking about the .gifs. I'm not certain what is causing this problem. The Create an Account .gif in the Getting Started section (l.75 of the Markdown) still works as it should.

I [noticed](https://github.com/programminghistorian/ph-submissions/issues/426#issuecomment-1213892420) that the tiny icon images no longer displayed 'inline' with the text when I added in the liquid syntax. If we want to keep the icons, and display them inline, I think we might need to add the `text` formatting back in.

I know that liquid syntax _can_ work with .gifs because I checked the code for an ES lesson that I remembered includes them: [Análisis de corpus con Voyant Tools](https://programminghistorian.org/es/lecciones/analisis-voyant-tools). The Markdown for that lesson is [here](https://github.com/programminghistorian/jekyll/edit/gh-pages/es/lecciones/analisis-voyant-tools.md).

--

Reviewing [my comment in the Issue thread](https://github.com/programminghistorian/ph-submissions/issues/426#issuecomment-1213892420) and double checking the recent commit history, it looks like we are still waiting for the authors to make the other small changes, or reply with their responses to those suggestions.

**Comment by hawc2** [comment]
Ok, thanks @anisa-hawes  - I'll wait until the final edits are made based on our discussion on the lesson ticket. After they finalize the lesson on ph-submissions, I'll update this branch, and then we can hopefully go forward with this pull request

**Comment by hawc2** [comment]
@svmelton, the authors are still trying to figure out how to edit the gifs to render correctly. Should we close this and I can start a new pull request when it's ready?

**Comment by svmelton** [comment]
> @svmelton, the authors are still trying to figure out how to edit the gifs to render correctly. Should we close this and I can start a new pull request when it's ready?

Yes, that sounds good!

**Pull request closed without merging**